api/tests/src/api/opportunities_v1/test_opportunity_route_search.py

```diff
@@ -1305,3 +1305,55 @@ def test_search_query_facets_200(self, client, api_auth_token):
             "funding_category",
             "opportunity_status",
         }
+
+    @pytest.mark.parametrize(
+        "search_request,expected_response",
+        [
+            # default scoring rule
+            (
+                get_search_request(
+                    order_by="opportunity_id",
+                    sort_direction=SortDirection.ASCENDING,
```

Comment:
If you wanted to make these tests even more minimal, the order_by and sort_direction can be dropped here as the method adds some defaults to make a valid request.